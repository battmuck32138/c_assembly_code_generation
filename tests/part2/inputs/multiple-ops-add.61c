int main () {
  return -(-4 + 6) + -(-7 + -2);
}

