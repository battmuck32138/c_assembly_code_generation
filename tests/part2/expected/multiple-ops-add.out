
Exited with error code 7
